Mathlib/Combinatorics/Digraph/Basic.lean

```diff
@@ -0,0 +1,252 @@
+/-
+Copyright (c) 2024 Kyle Miller, Jack Cheverton. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Kyle Miller, Jack Cheverton
+-/
+import Mathlib.Order.CompleteBooleanAlgebra
+import Mathlib.Data.Fintype.Pi
+import Mathlib.Tactic.ApplyFun
+
+/-!
+# Digraphs
+
+This module defines directed graphs on a vertex type `V`.
+
+## Main definitions
+
+* `Digraph` is a structure for the relation
```

Comment:
This bullet point doesn't convey much to me. You can't say "the relation" without the prior text giving some indication which relation!